src/cairo_interaction.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2013-2015, The Rust-GNOME Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+//! Functions to support using cairo
+
+use std::mem;
+use glib::translate::*;
+use ffi::{self, GdkRGBA};
+use super::{Pixbuf, Window};
+use cairo::{Context, RectangleInt};
+
+pub fn create(window: &Window) -> Context {
```

Comment:
We could do that, but there are other functions on the [documentation page](https://developer.gnome.org/gdk3/stable/gdk3-Cairo-Interaction.html) like `create_region_from_surface()` and `create_surface_from_pixbuf()` which don't really fit in as extensions of `Context`.  I commented them out for now, because regions haven't been wrapped, but it's conceivable that they'll be added in the future.